bin-src/lib/getStorybookInfo.ts

```diff
@@ -35,7 +37,8 @@ const findViewlayer = async ({ env, log, options, packageJson }) => {
   // Allow setting Storybook version via CHROMATIC_STORYBOOK_VERSION='@storybook/react@4.0-alpha.8' for unusual cases
   if (env.CHROMATIC_STORYBOOK_VERSION) {
     const [, p, v] = env.CHROMATIC_STORYBOOK_VERSION.match(/(.+)@(.+)$/) || [];
-    const version = semver.valid(v); // ensures we get a specific version, not a range
+    const version = semver.valid(v) || semver.validRange(v); // ensures we get a specific version, not a range
```

Comment:
Can we update this comment because I think it now might return a range right?

Reply:
Actually, I'll revert this change. Understanding the context of using this flag, it's to explicitly use a specific version of Storybook.

Reply:
@JonathanKolnik I have reverted this change!